is.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   array: Array.isArray,
-  primitive: function(s) { return typeof s === 'string' || typeof s === 'number'; },
+  primitive: function(s) {
+      return typeof s === 'string' || typeof s === 'number' || typeof s === 'boolean';
```

Comment:
This still doesn't cover all primitive values (e.g. 'undefined', 'symbol') and returns `false` for `null`.

To check if the value is primitive you should check that it's not an object, like so: `(x) => !isObject(x)`, where `isObject` is something like [`_.isObject from lodash`](https://github.com/lodash/lodash/blob/8176d56c725489156e19d68aeaf8851d313f4446/dist/lodash.core.js#L2699).